jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -219,4 +221,65 @@ public String getName() {
   public String getVersion() {
     return project.getVersion();
   }
+
+  void validateBaseImageVersion(@Nullable String baseImage) throws MojoFailureException {
+    if (!PluginConfigurationProcessor.usingDefaultBaseImage(baseImage)) {
```

Comment:
Could we anticipate having a JVM version label and change this to be something like:
```
  Optional<JvmVersion> getBaseImageJvmVersion(@Nullable String image);
  Optional<JvmVersion> getBuildJvmVersion();

  public void validateBaseImageVersion(@Nullable String baseImage) {
    …
  }
```
And we can check now if we have `distroless/java:8` or `:11`.

Reply:
Maybe I'm not totally sure what you mean, but is this something that's currently used? If it's not currently in practice I think we can hold off on it for now.

Reply:
I was understanding it in the same way, @TadCordle, so I agree we can hold off for now.